server/py/services/api/utils/db/partitioner.py

```diff
@@ -46,9 +47,12 @@ def create_and_drop_partitions(
             table_name,
         )
 
-        # Ensure partitions for double the retention time.
+        # Ensure partitions for the retention time plus
+        # mlrun.config.partitions_buffer_multiplier * partitioned_interval
```

Comment:
no need to write the math twice
```suggestion
        # Ensure partitions for the retention time plus a buffer
```